crates/goose/src/agents/capabilities.rs

```diff
@@ -33,7 +33,7 @@ impl Capabilities {
     /// Add a new MCP system based on the provided client type
     // TODO IMPORTANT need to ensure this times out if the system command is broken!
     pub async fn add_system(&mut self, config: SystemConfig) -> SystemResult<()> {
-        let client: McpClient = match config {
+        let mut client: McpClient = match config {
```

Comment:
does this need to be mut as no other changes?

Reply:
this came from the merge of https://github.com/block/goose/pull/558